### ISSUE TITLE
Can no longer change user settings

### ISSUE THREAD

**Issue by muff1nman** [Libresonic/libresonic#179]
Getting a 400 on post to user settings controller. I'll continue to investigate, however I think we need better exception handling. Its troubling that we would send a 400 but there be no mention of it in the logs..

**Comment by alexfornuto** [comment]
I just started a fresh install from the prepackaged war file, and got this error when trying to change the default admin password. I found this in `/var/libresonic/db/libresonic.log ` :

```
CONNECT USER SA
DISCONNECT
/*C616*/SET SCHEMA PUBLIC
CONNECT USER SA
/*C617*/SET SCHEMA PUBLIC
CONNECT USER SA
DISCONNECT
```

A lot of ^ that.

**Comment by muff1nman** [comment]
I don't believe that is related. What I found was that the user was not getting selected properly due to different behavior between the deprecated AbstractFormController and how spring now treats SessionAttributes.

Currently debating on whether we should even use SessionAttributes in this case

**Comment by alexfornuto** [comment]
I've reopened this issue because the latest builds still have this issue. I still can't change the administrator password on a fresh install.

**Comment by biconou** [comment]
During Spring 4 migration I removed the password controller. I don't know if it's related to this issue. I'll take a look after holidays

**Comment by alexfornuto** [comment]
@biconou if you can tell me which commit that was, I'll test around it.

**Comment by biconou** [comment]
@alexfornuto can you describe exactly your test case ? 
I just tested from develop branch and I managed to change the administrator password.

**Comment by muff1nman** [comment]
I am working on fixing this

**Comment by muff1nman** [comment]
Actually I think @biconou was right in that it should be working. The thing that I had run into is that if there any form errors, you will get a 400 error, basically #180. But now that #180 is fixed, I would be interested to see if you still run into issues.
